packages/mirinae/src/inputs/dropdown/select-dropdown/PSelectDropdown.vue

```diff
@@ -271,11 +273,14 @@ watch(() => props.handler, async () => {
                     return item;
                 });
             } else {
-                console.error(`Failed to fetch data from handler: ${idx}`);
+                throw new Error(`Failed to fetch data from handler: ${idx}`);
             }
         });
```

Comment:
issue: If error is thrown here, if there are errors on handler execution, this component will be stopped 🥲
I think that would be critical..
What about to change lie below:
```typescript
console.error(new Error(`Failed to fetch data from handler: ${idx}`))
```